### DISCUSSION TITLE
Lsposed is being detected

### DISCUSSION THREAD

**Discussion by ItsMadruga** [mywalkb/LSPosed_mod#40] [Q&A] [answered]
### Steps to reproduce/复现步骤

1.instalar o lsposed_mod
2.start the Inter&Co application

### Expected behaviour/预期行为

Use the Application normally, without lsposed being detected.

### Actual behaviour/实际行为

The application says that my device is not approved and closes, if you disable LsPosed, it works normally

### Xposed Module List/Xposed 模块列表

```shell
Bootloader Spoof
MdgWa
Firefds Kit
Knox Patch
HideMyAppList
```

### Magisk Module List/Magisk 模块列表

```shell
Knox Patch
Play Integrity Fix
Sensitive Props
Lsposed_mod
Shamiko
ZygiskNext
```

### LSPosed version/LSPosed 版本

7242

### Android version/Android 版本

14

### Magisk version/Magisk 版本

KernelSu 11563 v0.8.1-20

### Riru version/Riru 版本

n/a

### Version requirement/版本要求

- [x] I am using latest debug CI version of LSPosed and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Logs/日志

[LSPosed_2024-03-05T01_36_15.646841.zip](https://github.com/mywalkb/LSPosed_mod/files/14490398/LSPosed_2024-03-05T01_36_15.646841.zip)

**Comment by ghost** [comment]
Skill Issue

![Screenshot_20240305-151232_Trebuchet](https://github.com/mywalkb/LSPosed_mod/assets/85633117/cc4ddaa1-ff74-4d80-94f6-3306fd5f1954)

**Comment by ItsMadruga** [comment]
The problem doesn't appear if you don't already have an account, so much so that I managed to log in 

![Screenshot_20240305_101429_One UI Home](https://github.com/mywalkb/LSPosed_mod/assets/104334266/380416d2-b1c8-414f-a3dd-0cd381a7e370)

**Reply by ItsMadruga** [reply]
@mywalkb Do you have any idea what it could be?

**Reply by mywalkb** [reply]
they almost certainly used a commercial library like appdome which detects root (Magisk and similar) and Xposed (LSPosed and similar). 
They use quite sophisticated techniques, the shared libraries are obfuscated so it is not easy to reverse engineering, in the past I tried to find a solution but without success. I think the only way is to exclude that app from the scope of any module.

**Reply by ItsMadruga** [reply]
ha is excluded :(

**Reply by mywalkb** [reply]
strange because lsposed is only detected if the process is modified for hooks.
I tried to install the app and moving on, I don't know portuguese so I can't do much, but there's no alert message.
I put the app in denylist, I don't have shamiko but just my denylistunmount module which is very simple.
Try to clear cache of the app.

**Reply by mywalkb** [reply]
I maked an account, so it should work also for you.

**Reply by ItsMadruga** [reply]
I made your suggestions, but it didn't work

**Reply by mywalkb** [reply, marked as answer]
I analyzed your logs, the app is hooked by module es.chiteroman.bootloaderspoofer, if you remove the app from that module should be work

**Reply by juvannx** [reply]
have you solved? I have a similar issue with another app
